codegen/src/generator/tests.rs

```diff
@@ -16,10 +16,10 @@ const OUR_TESTS_DIR: &'static str = concat!(env!("CARGO_MANIFEST_DIR"), "/tests/
 pub fn generate_tests(writer: &mut FileWriter, service: &Service) -> IoResult {
     writeln!(writer,
              "
-            #[cfg(test)]
-            mod protocol_tests {{
-                {tests_body}
-            }}
+#[cfg(test)]
```

Comment:
I've been deliberately indenting the generated code to match the indentation level of the surrounding rusoto code, as it's the only code that will be read by humans.  i.e., indentation in the generated code doesn't matter because nobody will look at it, but it's way more readable if it's indented here.